seqr/views/utils/export_utils_2_3_tests.py

```diff
@@ -85,7 +88,20 @@ def mock_write_zip(file, content):
         })
         mock_zip_content = {}
 
+        # test unicode chars in zip filename
```

Comment:
you don't need like a separate test case for this, just modify the file name in the above test to have unicode

Reply:
okay